runner/src/tockloader.rs

```diff
@@ -31,9 +32,10 @@ pub fn deploy(cli: &Cli, platform: String, tab_path: PathBuf) -> Child {
     // varies from platform to platform. We look up the platform, and if it is
     // not satisfactorily reliable we output a warning for the user.
     let reliable_listen = match platform.as_str() {
-        // tockloader listen will reset the Hail, allowing it to capture all
+        // tockloader listen will reset the Hail/Imix, allowing it to capture all
         // printed messages.
         "hail" => true,
+        "imix" => true,
```

Comment:
Nitpick: use `"hail" | "imix" => true,` to make this consistent with the `"nrf52" | "nrf52840" => false,` case below.

Reply:
You changed something in this file, but I think you forgot to push the change that addressed my comment.

Reply:
Ah, my mistake! Should actually be fixed now.